### ISSUE TITLE
please tag a release (at least a bugfix one)

### ISSUE THREAD

**Issue by tpapp** [Jutho/KrylovKit.jl#54]
Otherwise the package does not work on 1.8 or master, lacking ae53a892a7c2e1804fbb55978ee2d056c2155cb.

**Comment by Jutho** [comment]
My apologies, I was traveling this week. I'll try to have a look early next week. As you might have noticed, I am in the middle of preparing a larger update (including preliminary AD support) but still have quite a bit of work for that. I'll try to find a nice way to already tag a bugfix release.

**Comment by tpapp** [comment]
No worries. My expectation is that backporting
```julia
@static if VERSION >= v"1.7"
    const liblapack = LinearAlgebra.BLAS.libblastrampoline
else
    const liblapack = LinearAlgebra.LAPACK.liblapack
end
```
to the latest tagged release would work, but I didn't check it.

**Comment by Jutho** [comment]
0.5.4 should now be available, 0.6 with preliminary partial AD support will hopefully follow soon
